tests/pages/author/design-questionnaire.page.js

```diff
@@ -1,34 +1,34 @@
 class DesignQuestionnaire {
   static setSectionTitle() {
-    return "#section-editor [aria-label='title']";
+    return "[data-testid='txt-section-title']";
   }
 
   static setQuestionTitle() {
-    return "#question-page-editor [aria-label='Question']";
+    return "[data-testid='txt-question-title']";
```

Comment:
why do we have some as `data-test` and some as `data-testid`?

Reply:
Because draftjs only renders certain predefined attributes to the dom, you have to send it a test attribute using `webDriverTestID` and then that prop gets reassigned and outputted to the dom as data-testid.

Reply:
How does that cause inconsistencies?

Reply:
Whenever I want to reference a draftjs editor I have to use its test attributes rather than my own. Its test attribute is always outputted as data-testid so thats what I have to reference.